src/olympia/bandwagon/views.py

```diff
@@ -150,6 +150,18 @@ def list(self, request, *args, **kwargs):
             content_type='application/json',
         )
 
+    def list(self, request, *args, **kwargs):
+        # This endpoint can be quite slow so we cache the most popular
+        # collections - those from mozilla - for all anonymous users for one
+        # hour.
+        if (
+            self.kwargs['user_pk'] in ('mozilla', str(settings.TASK_USER_ID))
+            and not request.user.is_authenticated
+        ):
+            return self._cached_list(request, *args, **kwargs)
+        else:
+            return super().list(*args, *kwargs)
```

Comment:
Why do we want to strip `request` from the args/kwargs?

Reply:
Good eye, that's a mistake. It fortunately doesn't have any consequences, DRF's default implementation for `list()` doesn't use that parameter or any other `args`, but I'll fix it.